js/testimonial.js

```diff
@@ -0,0 +1,23 @@
+
+var CURRENT_TESTIMONIAL = 0;
```

Comment:
this is not a constant, pls use lower camel case and better wrap it with namespace